TicTacToeProject/MinimaxStrategy.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace TicTacToeApp
+{
+    public class MinimaxStrategy : IStrategy
+    {
+        public int bestMove;
+        public Board originalBoard;
+        public Rules rules;
+        public string currentPlayerMarker;
+        public string opponentPlayerMarker;
+
+        public MinimaxStrategy(Board originalBoard)
+        {
+            this.rules = new Rules();            
+            this.originalBoard = originalBoard;    
+        }
+
+        public void setPlayers()
+        {
+            var current = originalBoard.getCurrentPlayer();
+            if (current == Symbols.P1_MARKER)
+            {
+                currentPlayerMarker = current;
+                opponentPlayerMarker = Symbols.P2_MARKER;
+            }
+            else
+            {
+                currentPlayerMarker = Symbols.P2_MARKER;
+                opponentPlayerMarker = Symbols.P1_MARKER;
+            }
+        }
+
+        public string getMove()
+        {
+            setPlayers();            
+            var depth = 0;
+            var score = minimax(originalBoard, currentPlayerMarker, opponentPlayerMarker, depth);
+            Console.WriteLine("\nThe minimax computer selected space {0}.", bestMove);
+            return bestMove.ToString();
+        }
+
+        public int score(Board board, string playerMarker, int depth)
+        {
+            bool isWon = rules.checkIfWon(board.board, playerMarker);
```

Comment:
Maybe rename `board.board` to `board.spaces`?
So in the Board.cs file you can change line 8:
from: `public List<Space> board = new List<Space>(); `
to: `public List<Space> spaces = new List<Space>(); `